src/main/java/mekanism/common/content/gear/mekasuit/ModuleJetpackUnit.java

```diff
@@ -55,4 +61,34 @@ public void changeMode(IModule<ModuleJetpackUnit> module, Player player, ItemSta
     public JetpackMode getMode() {
         return jetpackMode.get();
     }
+
+    public float getThrustMultiplier() {
+        return thrustMultiplier.get().getMultiplier();
+    }
+
+    @NothingNullByDefault
+    public enum ThrustMultiplier implements IHasTextComponent {
+        HALF(.5f),
+        NORMAL(1f),
+        FAST(2f),
+        FASTER(4f),
+        FASTEST(8f);
```

Comment:
I feel like rather than making this be based on how many are installed (or maybe in addition to doing so), we may want to increase the fuel usage for higher than normal thrust values.

Reply:
I like that idea. A potential issue there is that increasing the size of the hydrogen tank based on the number of modules isn't trivial. I know it's not necessary since at the same speed it would be able to go as far, but it does feel weird not to increase if it'll affect the time at max speed. (not that it will ever actually come up with the breathing unit refilling it)

Reply:
I added the dynamic fuel usage. Since it also affected hover mode I changed hover to increase/decrease the ascension speed based on the thrust. I was surprised how fast the 8x used up the fuel so we might just want the usage to increase by 1 each tier instead of doubling with the thrust values if we can't increase the tank size.

Reply:
The `gasTankSpec` takes a `LongSupplier`. It should be simple enough to change that to a `ToLongFunction` that takes an `ItemStack`, and then in `ChemicalTankSpec#createTank` turn it into a long supplier using the stack that is passed in.

Reply:
I went ahead and pushed a commit that makes the tank size scale with the number of installed modules, and handles the case where the thrust usage would be higher than the amount of remaining gas (even if it is only a single usage so doesn't fully matter)